config.json

```diff
@@ -162,10 +162,12 @@
       ],
       "namespaces": {
         "Admin": true,
+        "Audit": true,
```

Comment:
I believe @tompersona was looking at audit, and generally defer to him on when we want to expose these.